parlai/crowdsourcing/utils/tests.py

```diff
@@ -431,6 +430,16 @@ def _send_agent_message(
         self.await_channel_requests()
 
 
+def collapse_whitespace(in_string: str):
```

Comment:
oh ha, this is great to have, I'm sure you had to deal with some frustrating table output check issues :/

Reply:
Haha it wasn't too bad - I saw a few tests fail, copied and pasted the lines, and saw it was only whitespace. Probably just using a different version of `tabulate`